spec/features/viewing_taxons_spec.rb

```diff
@@ -321,6 +321,21 @@ def then_i_do_not_see_the_number_of_email_subscribers
   end
 
   def and_i_can_download_the_taxonomy_in_csv_form
+    document_counts = {
+      'facets' => {
+        'taxons' => {
+          'options' => [
+            { 'value' => { 'slug' => 'Root' }, 'documents' => 1 },
+            { 'value' => { 'slug' => 'Child-1' }, 'documents' => 2 },
+            { 'value' => { 'slug' => 'Child-2' }, 'documents' => 3 },
+            { 'value' => { 'slug' => 'Child-3' }, 'documents' => 4 },
+          ]
+        }
+      }
+    }
+
+    stub_request(:get, "https://search.test.gov.uk/search.json?count=0&facet_taxons=4000")
```

Comment:
some pedantic stuff, but you could use ```.with(query: {"count" => 0, "facet_taxons"=>4000})``` in the stub, that way the order of the parameters don't matter.